CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Ewon Ignition 8 Connector Changelog
 
+## v1.2.1
+
+### Major Changes
+
+* Fixed an issue that caused the per-tag realtime override option to be missing
+  * Tag provider name customization was removed to ensure consistent and stable connector functionality

```

Comment:
Why does that make it more consistent and stable?

Reply:
If you change the name of the tag provider, custom tag options become unstable. For example, the per-tag realtime override option will never be populated on tags present before a tag provider name change.

Reply:
Can you document that in the commit body for fb3c30a5e43528e386fb33d037beb483e1425503? Right now there really any reason given why its unstable.

Reply:
Why do you feel that adding more details to the commit message is redundant?

The readme is very user focused while the commit messages should be the documentation for developers. 

If i were to pick up this project and saw:
`Remove provider name customization

Removing the customization of the tag
provider name allows for more stable
and consistent functionality of the
connector.`

I really wouldnt have much of a clue as to why removing the ability to customized made it more stable.

Reply:
Adding more details would not be redundant, but the relevant detail/explanation is already in the other bullet point, 

```
* Fixed an issue that caused the per-tag realtime override option to be missing
  * Tag provider name customization was removed to ensure consistent and stable connector functionality
```

Reply:
Im not sure if im being clear enough to get my point across. I have no problem with the changelog the way it is now. 

My concern is that the commit message for fb3c30a5e43528e386fb33d037beb483e1425503 does not provide any details to future developers on why the tag provider customization causes instability/consistency issues in the application. The developer would expect to find detailed information about this in the commit message rather than in the public facing changelog notes for the release. I dont think the changelog should be more clear as to the reason for a commit than the commit message itself. Also we shouldn't fear repeating information between commit messages and changelogs.

Reply:
OHH, the commit message. 

Sorry, I overlooked that in your message earlier and confused myself.

Reply:
Pushed an updated commit message.